### PR TITLE
[FLINK-6715] Activate strict checkstyle for flink-mesos

### PR DESCRIPTION
**Pull request by zentol**

### CODE REVIEW

File: flink-mesos/src/main/java/org/apache/flink/mesos/runtime/clusterframework/LaunchableMesosWorker.java [hunk 1]
```diff
@@ -284,8 +286,8 @@ public String toString() {
 				break;
 
 			case DOCKER:
-				assert(params.containerImageName().isDefined());
-					containerInfo
+				assert (params.containerImageName().isDefined());
```
Comment (greghogan) [inline flink-mesos/src/main/java/org/apache/flink/mesos/runtime/clusterframework/LaunchableMesosWorker.java]: Was this change required / intended?

Comment (zentol) [inline flink-mesos/src/main/java/org/apache/flink/mesos/runtime/clusterframework/LaunchableMesosWorker.java]: hmm. not sure, I'll double check later.

Comment (zentol) [inline flink-mesos/src/main/java/org/apache/flink/mesos/runtime/clusterframework/LaunchableMesosWorker.java]: Change is not required, will revert it while merging.
